src/components/TaskCard/TaskCard.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import formatDate from "@helpers/formatting/formatDate";
+import "@components/TaskList/TaskList.scss";
+
+function TaskCard({ task }) {
+    const { title, createdAt } = task;
+
+    return (
+        <div className="task-card">
+            <h2 className="task-card__title">{title}</h2>
+            <span>Created At: {formatDate(createdAt)}</span>
+        </div>
+    );
+}
+
+TaskCard.defaultProps = {
+    task: {},
+};
+
+TaskCard.propTypes = {
+    task: PropTypes.shape({
+        title: PropTypes.string,
+        createdAt: PropTypes.string,
+    }),
+};
```

Comment:
I think there should not be any use case where we call `TaskCard` without `task` as prop. Thus me may consider doing this change?

```suggestion
TaskCard.propTypes = {
    task: PropTypes.shape({
        title: PropTypes.string,
        createdAt: PropTypes.string,
    }).isRequired,
};
```